source/PyMaterialX/PyMaterialXGenShader/PyHwShaderGenerator.cpp

```diff
@@ -0,0 +1,34 @@
+//
+// TM & (c) 2019 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <PyMaterialX/PyMaterialX.h>
+
+#include <MaterialXGenShader/GenContext.h>
+#include <MaterialXGenShader/HwShaderGenerator.h>
+
+#include <string>
+
+namespace py = pybind11;
+namespace mx = MaterialX;
+
+void bindPyHwShaderGenerator(py::module& mod)
+{
+    mod.attr("VERTEX_STAGE") = mx::Stage::VERTEX;
+
+    mod.attr("VERTEX_INPUTS") = mx::HW::VERTEX_INPUTS;
```

Comment:
For consistency with the OSL constants being prefixed with "OSL_" above maybe these HW constants should be prefixed with "HW_"?